dashboard/client/src/service/log.ts

```diff
@@ -49,3 +49,15 @@ export const getLogDetail = async (url: string) => {
 
   return rsp.data as string;
 };
+
+export const getStateApiDownloadLogUrl = (nodeId: string, fileName: string) =>
```

Comment:
Can we have one file that centralizes all the state API calls? This makes it easier to check which file needs to be changed if our API changes

Reply:
I don't really want to change the existing pattern. We currently group by feature, not underlying api implementation.